src/DefinitionResolver.php

```diff
@@ -446,7 +446,7 @@ public function resolveExpressionNodeToType(Node\Expr $expr): Type
                 // Cannot get type for dynamic function call
                 return new Types\Mixed;
             }
-            $fqn = (string)($expr->getAttribute('namespacedName') ?? $expr->name);
+            $fqn = (string)($expr->getAttribute('namespacedName') ?? $expr->name . '()');
```

Comment:
Pretty sure `namespacedName` would need to get the `()` too, so you have to move the parenthesis